### ISSUE TITLE
Crash in ServiceInfo.py when no service is playing

### ISSUE THREAD

**Issue by technic** [openatv/enigma2#1921]
Hi there,

when `refstr` equals to "N/A" this line will crash with

https://github.com/openatv/enigma2/blob/62850fc50e92051a267335dce342d2103488d98e/lib/python/Screens/ServiceInfo.py#L181

```
File "/usr/lib/enigma2/python/Screens/ServiceInfo.py", line 181, in information
IndexError: list index out of range
```

also instead of this `split` tricks you can consider just using `getPath()` method of `eServiceReference`.

Kind regards,
Alex

**Comment by jbleyel** [comment]
This is strange because line 180 has the exact same split.

**Comment by jbleyel** [comment]
I what case do you have N/A as refstr?

**Comment by technic** [comment]
I think it is when no service is playing. I.e. before I had setup up any
sat or cable services.

**Comment by jbleyel** [comment]
This part of ServiceInfo.py is fixed in 6.5 already.

**Comment by technic** [comment]
Thx for quick fix!
